packages/zeal/README.md

```diff
@@ -0,0 +1,30 @@
+# @web3-onboard/zeal
+
+## Wallet module for connecting Zeal to web3-onboard
+
+See [Zeal](https://www.zeal.app/) for details
```

Comment:
Please add more details here as this is usually first contact for developers

Reply:
:heavy_check_mark: